AutoMaxLair.py

```diff
@@ -54,9 +55,15 @@
 def join(inst):
     """Join a Dynamax Adventure and choose a Pokemon."""
     inst.log('Run #' + str(inst.runs + 1) + ' started!')
-    inst.push_buttons((b'b', 2), (b'a', 0.5), (b'a', 1), (b'a', 1.5), (b'a', 1.5), (b'a', 1.5), (b'a', 1.5), (b'a', 1), (b'a', 1.5), (b'a', 4), (b'v', 1), (b'a', 5))
+    inst.push_buttons((b'b', 2), (b'a', 1), (b'a', 1.5), (b'a', 1.5), (b'a', 1.5), (b'b', 1))

```

Comment:
I needed to add a few changes to button press here to not miss the selection of legendary path here. I'm not totally sure how it behaves on other languages